README.md

```diff
@@ -230,6 +230,27 @@ returned, only the diff.
 
 Currently accepts the same options as the read stream except for reverse.
 
+#### `watcher = db.watch([range], [onchange])`
+
+Listens to changes that are on the optional `range`.
+
+`range` options are the same as `createReadStream` except for reverse.
+
+`watcher.watch()`\
+Starts watching for changes. By default it already does it. This is only useful after you `unwatch()`.
+
+`watcher.unwatch()`\
```

Comment:
Do you have a usecase in mind for unwatching? On first sight, it seems like it could also be done by destroying the watcher and creating a new watcher later

Reply:
I took the idea from `udx` actually, and it was easy to do the change

Reply:
But I agree it could be removed on this case

Reply:
Also I agree even more because `unwatch()` is what introduced `watching` property, so maybe it's good if we just remove unwatch lol more simple